README.md

```diff
@@ -156,3 +156,31 @@ title: Don't repeat yourself
 
 # {{ title }}
 ```
+
+### Shortcodes
+
+Shortcodes are reusable code snippets that allow us to sweep away complicated scripts into a nice, easy user interface.
+
+For example this is the script for a YouTube embed.
```

Comment:
```suggestion
#### YouTube video player embed
```

What if we have a sub-section per shortcode? As expect we might have more in future

Reply:
Yeah, good idea.